cpp/src/io/json/json_gpu.h

```diff
@@ -16,14 +16,15 @@
 
 #pragma once
 
-#include <cudf/table/table_device_view.cuh>
-#include <cudf/types.hpp>
+#include "io/json/json_common.h"
```

Comment:
I think this should also use `< >` if only to just be consistent with the other `include <io...>` statements.